guides/source/active_record_encryption.md

```diff
@@ -342,7 +487,7 @@ Key providers implement key management strategies. You can configure key provide
 
 #### DerivedSecretKeyProvider
 
-A key provider that will serve keys derived from the provided passwords using PBKDF2.
+The [`DerivedSecretKeyProvider`](https://api.rubyonrails.org/classes/ActiveRecord/Encryption/DerivedSecretKeyProvider.html) key provider serves keys derived from the provided passwords using PBKDF2. This is the key provider configured by default.
```

Comment:
For the next key providers only the provider name is used ("XKeyProvider" instead of "XKeyPRovider key provider"):
```suggestion
The [`DerivedSecretKeyProvider`](https://api.rubyonrails.org/classes/ActiveRecord/Encryption/DerivedSecretKeyProvider.html) serves keys derived from the provided passwords using PBKDF2. This is the key provider configured by default.
```